### ISSUE TITLE
[Bug]: tsserver language features stop working when formatter or linter added

### ISSUE THREAD

**Issue by torkelo** [LunarVim/LunarVim#1764]
### Problem description

tsserver language features work great initially, go to defintion, references, highlights same type elsewhere in buffer when cursor is over type. 

But as soon as I add this:
```lua
lvim.lang.typescript.formatters = { { exe = "prettier" } }
```

to config all the language features stop working (like go to defintion and all others). The language server seem to be connected and running: 

and LunarVim rolling updated today 

### Check the following

- [X] I am in a git managed directory
- [X] I restarted Neovim after running :LspInstall
- [X] I have a valid language configuration (~/.config/lvim/ftplugin/)
- [X] I checked the options (~/.config/lvim/config.lua)
- [X] The programs specified by the config are installed

### LunarVim version

rolling updated 2020-10-13

### Neovim version (>= 0.5)

v6-dev

### Relevant log output

```shell
Nothing relevant from lsp.log
```

### Screenshots

![Screenshot from 2021-10-13 11-58-30](https://user-images.githubusercontent.com/10999/137111820-12cc7192-b835-4d24-9148-daa987db64a8.png)

![Screenshot from 2021-10-13 12-07-31](https://user-images.githubusercontent.com/10999/137113324-90c6faba-3d1e-40a6-9bbd-c0e096fdae93.png)

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by kylo252** [comment]
I can't really reproduce, but maybe prettier is broken.
https://asciinema.org/a/3z9NjF2AnOo0N2qpckQivqAoj

can you try some other formatter other than prettier?

**Comment by torkelo** [comment]
Downgrading to neovim v0.5.1 fixed this problem for me

**Comment by torkelo** [comment]
Downgrading to neovim v0.5.1 fixed this problem for me

**Comment by vadimi** [comment]
Something like that happened to me as well, but with Go and Yaml language servers. All I did was `:LvimUpdate` when .go or .yaml file is open. After lvim restart none of the language features are working. I was able to track it down to the `ftplugin` regen issue - basically `:LvimUpdate` removes all files from `~/.local/share/lunarvim/site/after/ftplugin` and regenerates them, but it doesn't do it for the actively running lsp server. So if your language features are not working, check if `~/.local/share/lunarvim/site/after/ftplugin` contains the file for your language.
